service/matching/matching_engine_test.go

```diff
@@ -3015,6 +3434,22 @@ func (m *testTaskManager) DeleteTaskQueue(
 	return nil
 }
 
+// generateErrorRandomly states if a taskManager's operation should return an error or not
+func (m *testTaskManager) generateErrorRandomly() bool {
```

Comment:
in the future we could consider reusing the new fault-injection framework. for now this is fine